packages/application-components/src/components/form-main-page-layout/form-main-page-layout.tsx

```diff
@@ -0,0 +1,125 @@
+import type { FormEvent, ReactElement, ReactNode } from 'react';
+import { css } from '@emotion/react';
+import IconButton from '@commercetools-uikit/icon-button';
+import Avatar from '@commercetools-uikit/avatar';
+import Spacings from '@commercetools-uikit/spacings';
+import Text from '@commercetools-uikit/text';
+import { customProperties } from '@commercetools-uikit/design-system';
+import { warning } from '@commercetools-uikit/utils';
+import PageHeaderTitle from '../internals/page-header-title';
+
+export type TFormMainPageLayoutProps = {
+  /**
+   * The title of the page.
+   */
+  title?: string;
+  /**
+   * The title of the header.
+   */
+  headerTitle?: string;
+  /**
+   * The subtitle of the header.
+   */
+  headerSubtitle?: string | ReactElement;
+  /**
+   * Replaces the title/subtitle row with a custom one (for special use cases)
+   */
+  customTitleRow?: ReactNode;
+  /**
+   * Any React node displayed as the content within the page.
+   */
+  children: ReactNode;
+  /**
+   * The hashed string of the user gravatar.
+   */
+  gravatarHash?: string;
```

Comment:
I think there has been some misunderstanding here. These props/functionality should not be part of the component.
The component is only about the layout, like all the other layout pages.